src/Illuminate/Mail/Events/MessageSent.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Illuminate\Mail\Events;
+
+use Swift_Message;
+
+class MessageSent
+{
+    /**
+     * The Swift message instance.
+     *
+     * @var \Swift_Message
+     */
+    public $message;
+
+    /**
+     * The Swift transport.
+     *
+     * @var string
+     */
+    public $transport;
+
+    /**
+     * The Swift send result - format depends on the transport.
```

Comment:
Hmmm. The fact that this is true means we should probably omit the transport and result. I'd say it's a bad practice to mess with transport specifics. That's why we return void.